src/Components/Mobile_filter/Mobile_filter.jsx

```diff
@@ -104,17 +105,15 @@ export const Mobile_filter = () => {
           <div className="spacer"></div>
           <div className="flex-center-column ali-st gap-0-8">
             <div className="fnt-1 fnt-w-600">Ratings</div>
-            {ratings.map((ele) => (
-              <label
-                className="radio flex-center-row gap-0-5"
-                onChange={() => {
-                  setFilterState({
-                    type: "rating",
-                    paylod: ele,
-                  });
-                }}
-              >
+            {ratings.map((ele, index) => (
+              <label key={index} className="radio flex-center-row gap-0-5">
                 <input
+                  onChange={() => {
+                    setFilterState({
+                      type: "rating",
+                      paylod: ele,
+                    });
```

Comment:
suggestion: better to define type in capital letter .just a good practice

Reply:
Done ..